scripts/oe-env-checkout-branch

```diff
@@ -5,11 +5,14 @@ SRC="${BUILDDIR}/../../sources"
 
 pushd ${SRC}
 
-for i in $(ls)
-do
-  echo -e "\n>>> $i <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
-  pushd $i > /dev/null 2>&1
-  git fetch -a
-  git checkout $B
-  popd > /dev/null 2>&1;
+for i in *; do
+    if [ -d "$i" ]; then
+        echo -e "\n>>> $i <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
+        pushd $i > /dev/null 2>&1
```

Comment:
There's a simpler idiom for redirecting both `stderr` and `stdout` to `/dev/null`: `push $i &>/dev/null`

https://www.gnu.org/software/bash/manual/bash.html#Redirecting-Standard-Output-and-Standard-Error